tests/unit/spring_boot_patch.py

```diff
@@ -195,6 +195,7 @@ def __init__(self, original_container: ops.model.Container, image: OCIImageMock)
         self.file_system_mock = ContainerFileSystemMock(image=image)
         self.process_mock = ContainerProcessMock()
         self._original_container = original_container
+        self._pebble = original_container._pebble
```

Comment:
I don't where this is used

Reply:
This is needed behind for integration tests to pass since ops 2.0
Basically pebble is used by Harness at multiple places

Reply:
Well the whole module is not very intuitive and pretty complex in fact

Reply:
I added a minimal comment for the added line